### DISCUSSION TITLE
BEM (Build) with Wordpress

### DISCUSSION THREAD

**Discussion by marco-sandro-kraft** [getbem/getbem.github.io#261] [Q&A]
Hi there

Is it possible or recommended to create a wordpress-bootstrap-website with  BEM?
I have read the documentation and it makes sense to me to use this methodology.  But I am struggling to understand, how to apply the building part with wordpress.
Can you link me an FAQ or have some hints to continue

**Comment by iamstarkov** [comment]
you can take a look at this https://github.com/samhastings/wordpress-bem
